### PR TITLE
Add notifications 

### PR DESCRIPTION
**Pull request by GermanGolota**

### CODE REVIEW

File: src/Aurora.Application/Contracts/INotificator.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using Aurora.Application.Models;
+using System.Threading.Tasks;
+
+namespace Aurora.Application.Contracts
+{
+    public interface INotificator
```
Comment (RuslanPr0g) [inline src/Aurora.Application/Contracts/INotificator.cs:6]: Useless abstraction ?

File: src/Aurora.Presentation/Startup.cs [hunk 2]
```diff
@@ -44,6 +44,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseHangfireDashboard();
             }
 
+            app.UseAuthentication();
```
Comment (RuslanPr0g) [inline src/Aurora.Presentation/Startup.cs:47]: What about AddAuthorization() ???

**Review by RuslanPr0g — CHANGES_REQUESTED** [review]
What do you think?

File: src/Aurora.Presentation/Startup.cs (hunk 2 above)
Comment (RuslanPr0g) [reply src/Aurora.Presentation/Startup.cs:47]: called in diextentions

File: src/Aurora.Infrastructure/Services/SearchDataService.cs [hunk 3]
```diff
@@ -23,26 +22,28 @@ public async Task StoreRequest(SearchRequestDto request)
         {
             List<SearchRequest> existingRequests = await GetExistingFor(request);
 
+            List<SearchRequest> newRequests = new List<SearchRequest>();
             foreach (var webSite in request.Websites)
             {
                 foreach (var option in request.SearchOptions)
                 {
                     if (existingRequests.Where(x => x.Website == webSite && x.ContentOption == option).Any() == false)
```
Comment (RuslanPr0g) [inline src/Aurora.Infrastructure/Services/SearchDataService.cs]: _if (existingRequests.Where(x => x.Website == webSite && x.ContentOption == option).Any() == false)_

too hard to understand, perhaps move the logic to a method?

**Review by RuslanPr0g — CHANGES_REQUESTED** [review]

File: src/Aurora.Application/Contracts/INotificator.cs (hunk 1 above)
Comment (GermanGolota) [reply src/Aurora.Application/Contracts/INotificator.cs:6]: Fixed

File: src/Aurora.Infrastructure/Services/SearchDataService.cs (hunk 3 above)
Comment (GermanGolota) [reply src/Aurora.Infrastructure/Services/SearchDataService.cs]: moved to method

**Review by RuslanPr0g — APPROVED** [review]

**Pull request merged**